utils/src/main/java/com/revenuecat/purchases/utils/Responses.kt

```diff
@@ -0,0 +1,184 @@
+package com.revenuecat.purchases.utils
+
+object Responses {
+    const val validFullPurchaserResponse = """
```

Comment:
This is using strings now. It's faster and more clear. I don't think it's a big deal.

Reply:
I think it's fine